File: WYNKWeb/src/app/Models/ViewModels/ReferralMaster.model.ts
import { Referral_Master } from '../referralmaster.model';
import { Referral_Patients_HDR } from '../ReferralPatientHDRWebModel';

export class ReferralMaster {


  ReferralMaster: Referral_Master = new Referral_Master();
  Referral: Referral_Patients_HDR = null;
  Refdetail: Array<Refdetail> = [];
}
export class Refdetail {

  Referral_Code = '';
  Referral_Name = '';
  Phone_No = '';
  Email_Id = '';
  Referral_Address1 = '';
  Referral_Address2 = '';
  Referral_Address3 = '';
  Contact_Person = '';
}

File: WYNK.Data/Model/ReferralMaster.Model.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Text;

namespace WYNK.Data.Model
{
    public class ReferralMasterS
    {
        [Key]
        public string REFERRAL_CODE { get; set; }
        public string REFERRAL_NAME { get; set; }
        public string REFERRAL_ADDRESS1 { get; set; }
        public string REFERRAL_ADDRESS2 { get; set; }
        public string REFERRAL_ADDRESS3 { get; set; }
        public string FAX { get; set; }
        public string PHONE_NO { get; set; }
        public string EMAIL_ID { get; set; }
        public string CONTACT_PERSON { get; set; }
        public string REF_TYPE { get; set; }
    }
}
